### ISSUE TITLE
Extension upgrade from 2.0.0-alpha.3 to 2.0.1

### ISSUE THREAD

**Issue by Naufon** [tds-fdw/tds_fdw#238]
Hello.

When I'm trying to upgrade an existing extension:
# alter extension tds_fdw update;
ERROR:  extension "tds_fdw" has no update path from version "2.0.0-alpha.3" to version "2.0.1"

But creating extension on the other database works fine:
# \dx
                                            List of installed extensions
  Name   | Version |   Schema   |                                    Description
---------+---------+------------+-----------------------------------------------------------------------------------
 plpgsql | 1.0     | pg_catalog | PL/pgSQL procedural language
 tds_fdw | 2.0.1   | public     | Foreign data wrapper for querying a TDS database (Sybase or Microsoft SQL Server)

Restart the postgres didn't helped.
Recreating the extension is not a best way because we have a tons of foreign tables.

Any ideas?

**Comment by TheRevenantStar** [comment]
I'd recommend dumping the foreign objects, dropping the extension, using CREATE EXTENSION, then re-add your objects.
There's not really any easy way to remedy this unless one of the maintainers wants to tackle it.

**Comment by Naufon** [comment]
> I'd recommend dumping the foreign objects, dropping the extension, using CREATE EXTENSION, then re-add your objects.
> There's not really any easy way to remedy this unless one of the maintainers wants to tackle it.

Yeah. This is a hardest way because in this case we'll need a quite big downtime for the load process.
But if there is no other solution - we'll following this way.

**Comment by TheRevenantStar** [comment]
You could ask Postgres to create another cluster and do it there, it would just run on a different port until you finish, then you can modify the configuration to use the default port.

Just make sure you sync any changes after the fact

**Comment by Naufon** [comment]
Ok. Thanks!

**Comment by juliogonzalez** [comment]
I will reopen this.

FMPOV tds_fdw should be able to migrate. I will have a quick look to see if I can fix it myself, but no promises.

Otherwise someone else from the community should have a look :-)

**Comment by StefanWetterActiware** [comment]
This should be closed, @GeoffMontee ?
